NEWS.md

```diff
@@ -22,6 +23,10 @@
   * make `get_cell_refs()`, `get_date_origin()`, `guess_col_type()`, and `write_file()` internal functions
   * make classes `styles_mgr()`, `wbSheetData`, `wbWorksheet`, `wbChartsheet`, `wbComment`, `wbHyperlink` internal
   
+## Refactoring
+
+* Cleanup / revisit documentation and vignettes (#682, @olivroy) 
```

Comment:
we probably should switch to this pkgdown(?) style for issues and pull requests. Though, ofc the hyperlinks only work in the GitHub news this way and not in the CRAN version of the file and as soon as someone forks this project all of the links will point to nothing.

Reply:
you mean adding the full link [681]^(https://github.com/janmarvin/openxlsx2/issues/681) ?